.env

```diff
@@ -1,38 +1,44 @@
+# For all variables, see https://docs.lightdash.com/self-host/customize-deployment/environment-variables
+
 PORT=8080
 
 SITE_URL=http://localhost:3000
 
-DOCKERFILE_TARGET=dev
-
-PGHOST=db-dev
+PGHOST=db
 PGPORT=5432
 PGUSER=postgres
-PGPASSWORD=password
 PGDATABASE=postgres
+PGPASSWORD="password"
+LIGHTDASH_SECRET="not very secret"
 
 SECURE_COOKIES=false
 TRUST_PROXY=false
-LIGHTDASH_SECRET='not very secret'
-
-RUDDERSTACK_WRITE_KEY=1vikeGadtB0Y0oRDFNL2Prdhkbp
-RUDDERSTACK_DATA_PLANE_URL=https://analytics.lightdash.com
-
-DBT_DEMO_DIR=/usr/app/examples/full-jaffle-shop-demo
+LIGHTDASH_LOG_LEVEL=debug
+LIGHTDASH_QUERY_MAX_LIMIT=5000
 
-SENTRY_DSN=''
 
-NODE_ENV=development
+NODE_ENV=production
```

Comment:
do we need to change this?

Reply:
Jose said this .env should represent production, so I think having NODE_ENV=production on .env makes sense to me